lib/components/util/icon.tsx

```diff
@@ -28,12 +32,20 @@ const FontAwesomeWithSpace = styled(FontAwesome)`
  */
 const Icon = ({
   fixedWidth = true,
+  intl,
   type,
   withSpace = false,
   ...props
 }: IconProps): JSX.Element => {
   const FontComponent = withSpace ? FontAwesomeWithSpace : FontAwesome
-  return <FontComponent fixedWidth={fixedWidth} name={type} {...props} />
+  return (
+    <FontComponent
+      aria-label={intl ? getFormattedMode(type, intl, true) : ''}
```

Comment:
Should we fallback to `type` if `intl` is absent?

Reply:
Yes this was very much a temporary patch. I'm not sure however what the "correct" solution looks like. What would the fallback be?